CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## [0.0.5-dev.4] - v0.0.5-dev.4
+
+* Update default user-agent
```

Comment:
Once you update by removing the user-agent changes, please help to update the changelog to express you update webview_flutter to version 3.x as well. Thank you @NatsuOnFire

Reply:
@NatsuOnFire can you help to update based on above suggestion?